src/durableorchestrationbindinginfo.ts

```diff
@@ -1,12 +1,25 @@
 import { HistoryEvent } from "./classes";
+import { UpperSchemaVersion } from "./replaySchema";
 
 /** @hidden */
 export class DurableOrchestrationBindingInfo {
+    public readonly upperSchemaVersion: UpperSchemaVersion;
+
     constructor(
         public readonly history: HistoryEvent[] = [],
         public readonly input?: unknown,
         public readonly instanceId: string = "",
         public readonly isReplaying: boolean = false,
-        public readonly parentInstanceId?: string // TODO: Implement entity locking // public readonly contextLocks?: EntityId[],
-    ) {}
+        public readonly parentInstanceId?: string,
+        upperSchemaVersion = 0 // TODO: Implement entity locking // public readonly contextLocks?: EntityId[],
+    ) {
+        // If the extension-sent upperSchemaVersion is within the range of values
+        // we support, we select it. Otherwise, we conclude it's higher than any
+        // version we support, so we default to our highest version.

```

Comment:
I'm having trouble understanding these comments because they seem contradictory. If `upperSchemaVersion` isn't specified, do we assume it's higher than any version we support or do we assume they're using the highest version?